TOMP-API.yaml

```diff
@@ -908,27 +913,27 @@ paths:
         - payment
         - MP
         - TO
-      description: Returns all the journal entries that should be payed per leg
+      description: Returns all the journal entries that should be paid per leg
       parameters:
         - name: from
-          in: path
+          in: query
```

Comment:
I see this hotfix code is not already in develop-minor, shall I merge that first?

Reply:
please do so